packages/effects/src/lib/request.utils.test.ts

```diff
@@ -181,7 +186,9 @@ describe('initQuery function', () => {
 
     // @ts-expect-error - Intentionally testing an unknown action type
     const queryApi = initQuery(fetchArgs, endpoint).applyMiddleware(requestMiddleware);
-    const response = await queryApi.applyQuery(async (fetchArgs) => await mockQuery(fetchArgs));
+    const response = await queryApi.applyQuery(
+      async (fetchArgs: FetchArgs) => await mockQuery(fetchArgs),
```

Comment:
for whatever reason, fetchArgs in my IDE was properly inferred, but the typecheck command was unhappy (i couldnt figure out at that time what was causing it though) worth looking into more though.